src/Utils/Jwks.php

```diff
@@ -47,6 +47,7 @@ private function createKey(string $certificate, $subject) : array
             JwkParameter::KEY_TYPE => 'RSA',
             RsaParameter::PUBLIC_EXPONENT => 'AQAB', // Hard-coded as `Base64Url::encode($keyInfo['rsa']['e'])` tends to be empty...
             RsaParameter::PUBLIC_MODULUS => Base64Url::encode($subject),
+			JwkParameter::KEY_OPERATIONS => array("verify")
```

Comment:
Even though this is the last item in the array, it is preferred to have a trailing comma, so future additions only change the line that is added instead of also changing the previous line...

Reply:
fixed that.